### PR TITLE
Wip unified cdc

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/configuration/CdcPipelineFactoryConfiguration.java [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 import io.eventuate.local.unified.cdc.pipeline.common.BinlogEntryReaderProvider;
 import io.eventuate.local.unified.cdc.pipeline.common.factory.CdcPipelineFactory;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-public class PollingCdcPipelineFactoryConfiguration {
-  @Bean("evenutateLocalPollingCdcPipelineFactory")
+public class CdcPipelineFactoryConfiguration {
+  @Bean("evenutateLocalСdcPipelineFactory")
```
Comment (cer) [inline new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/configuration/CdcPipelineFactoryConfiguration.java]: This is mispelt `evenutate...`

Comment (dartartem) [reply new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/configuration/CdcPipelineFactoryConfiguration.java]: fixed

File: new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/dblog/mysqlbinlog/factory/MySqlBinlogCdcPipelineReaderFactory.java [hunk 2]
```diff
@@ -1,41 +1,75 @@
...
-    return new DatabaseOffsetKafkaStore(properties.getDbHistoryTopicName(),
-            clientName,
-            eventuateKafkaProducer,
+  @Override
+  public Class<MySqlBinlogCdcPipelineReaderProperties> propertyClass() {
+    return MySqlBinlogCdcPipelineReaderProperties.class;
+  }
+
+  @Override
+  public MySqlBinaryLogClient create(MySqlBinlogCdcPipelineReaderProperties mySqlBinlogCdcPipelineReaderProperties) {
```
Comment (cer) [inline new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/dblog/mysqlbinlog/factory/MySqlBinlogCdcPipelineReaderFactory.java]: Please rename the parameter name from mySqlBinlogCdcPipelineReaderProperties to => readerProperties

File: new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/dblog/postgreswal/factory/PostgresWalCdcPipelineReaderFactory.java [hunk 3]
```diff
@@ -1,43 +1,66 @@
...
-            eventuateKafkaProducer,
-            eventuateKafkaConfigurationProperties,
-            eventuateKafkaConsumerConfigurationProperties);
+  @Override
+  public Class<PostgresWalCdcPipelineReaderProperties> propertyClass() {
+    return PostgresWalCdcPipelineReaderProperties.class;
   }
 
+  @Override
+  public PostgresWalClient create(PostgresWalCdcPipelineReaderProperties postgresWalCdcPipelineReaderProperties) {
```
Comment (cer) [inline new-cdc/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/dblog/postgreswal/factory/PostgresWalCdcPipelineReaderFactory.java]: Please rename parameter: postgresWalCdcPipelineReaderProperties => readerProperties

**Pull request merged**